src/components/boardComponents/useItemActions.jsx

```diff
@@ -109,18 +109,15 @@ export const useItemActions = () => {
         )
       );
 
-      const [newX, newY] = [
-        (minMax.min.x + minMax.max.x) / 2,
-        (minMax.min.y + minMax.max.y) / 2,
-      ];
+      const [newX, newY] = [minMax.min.x, minMax.min.y];
```

Comment:
What seems odd to me with the new version:
- if you stack distant items, you can have all your items stacked to the top left of selection instead of middle of the selection. I like the "gathering" effect it give when grouping to the center.
- I think it less looks like a stack with the descendant order
- We lost stack elevation precision on little number of stacked item

My guess, by seeing your video, is that you're trying to have predictible and stable card alignement ? If you stack then align then stack again with the previous version, the cards are stacked in the center of the line which is not really convenient. My proposition: stack each item from the position of the deeper element in the global element array, the one which is "under" all other selected elements, this way if you press the same alignement multiple times, the cards doesn't move and if you change alignment then come back to the first one the cards get back to their original position. Is that ok for you ? In this case, you should modify your other alignment function to start from the same element.

I think you also want to avoid too high stack. For this point I propose to decrease the index modification by some factor (0.5) if we have more than, let's say, 32 cards in the stack ? What do you think ?